components/cluster/pacemaker/Makefile

```diff
@@ -15,35 +15,39 @@
 #
```

Comment:
I guess you'll need to add your well-known
# Bogus dependency due to libssp                                                                                                     
REQUIRED_PACKAGES += $(GXX_RUNTIME_PKG) 

because I got several errors like this:
ker/build/manifest-i386-pacemaker.depend
/export/home/andreas/oi-userland/components/cluster/pacemaker/build/manifest-i386-pacemaker.depend has unresolved dependency '
    depend type=require fmri=__TBD \
        pkg.debug.depend.reason=usr/lib/amd64/libpacemaker.so.1.1.0 \
        pkg.debug.depend.type=elf \
        pkg.debug.depend.fullpath=lib/64/libssp.so.0 \
        pkg.debug.depend.fullpath=usr/gcc/7/lib/amd64/libssp.so.0 \
        pkg.debug.depend.fullpath=usr/gnu/lib/amd64/libssp.so.0 \
        pkg.debug.depend.fullpath=usr/openssl/1.1/lib/64/libssp.so.0'.